org.opent2t.sample.binaryswitch.superpopular/com.insteon.binaryswitch/js/tests/__snapshots__/unitTests.js.snap

```diff
@@ -0,0 +1,3 @@
+exports[`GetPlatform 1`] = `"{\"opent2t\":{\"schema\":\"org.opent2t.sample.binaryswitch.superpopular\",\"translator\":\"opent2t-translator-com-insteon-binaryswitch\",\"controlId\":\"Mock Device\"},\"pi\":\"44b0a5fb-f8a2-0160-0fae-34978b82a172\",\"mnmn\":\"Insteon\",\"mnmo\":\"Binary Switch (Generic)\",\"n\":\"Mock Device\",\"rt\":[\"org.opent2t.sample.binaryswitch.superpopular\"],\"entities\":[{\"n\":\"Mock Device\",\"icv\":\"core.1.1.0\",\"dmv\":\"res.1.1.0\",\"rt\":[\"oic.d.smartplug\"],\"di\":\"f9604075-1a64-498b-ae9b-7436a63721ba\",\"resources\":[{\"href\":\"/power\",\"rt\":[\"oic.r.switch.binary\"],\"if\":[\"oic.if.a\",\"oic.if.baseline\"]},{\"href\":\"/connectionStatus\",\"rt\":[\"oic.r.mode\"],\"if\":[\"oic.if.s\",\"oic.if.baseline\"]}]}]}"`;
+
+exports[`GetPlatformExpanded 1`] = `"{"opent2t":{"schema":"org.opent2t.sample.binaryswitch.superpopular","translator":"opent2t-translator-com-insteon-binaryswitch","controlId":"Mock Device"},"pi":"44b0a5fb-f8a2-0160-0fae-34978b82a172","mnmn":"Insteon","mnmo":"Binary Switch (Generic)","n":"Mock Device","rt":["org.opent2t.sample.binaryswitch.superpopular"],"entities":[{"n":"Mock Device","icv":"core.1.1.0","dmv":"res.1.1.0","rt":["oic.d.smartplug"],"di":"f9604075-1a64-498b-ae9b-7436a63721ba","resources":[{"href":"/power","rt":["oic.r.switch.binary"],"if":["oic.if.a","oic.if.baseline"],"id":"power","value":false},{"href":"/connectionStatus","rt":["oic.r.mode"],"if":["oic.if.s","oic.if.baseline"],"id":"connectionStatus","supportedModes":["online","offline","hidden","deleted"],"modes":["offline"]}]}]}"`;
```

Comment:
Yes, these are snapshots of our Mock device state that Ava can generate.  If the translator or the test data change, and affect the output of getPlatforms then the snapshot comparison will fail unless it is also updated to reflect the new info.

As we make modifications to the translators, we want to be able to verify that the platform scheme output from get(...) doesn't accidentally change.

On failure, Ava will automatically log a diff against the snapshot, and explain what can be done to check in an updated version if the change was intentional.